src/devices/Bmx280/samples/Bme280.sample.cs

```diff
@@ -23,53 +23,70 @@ static async Task Main(string[] args)
             //set this to the current sea level pressure in the area for correct altitude readings
             const double defaultSeaLevelPressure = 1033.00;
 
-            var i2cSettings = new I2cConnectionSettings(busId, Bme280.DefaultI2cAddress);
+            var i2cSettings = new I2cConnectionSettings(busId, Bme280.DefaultI2cAddress + 1);
```

Comment:
I did also find that the default I2C Address is incorrect. Do you mind changing the constant on the same PR to use 0x77 instead?

Reply:
Oops, didn't mean to commit that. I'll update the constant.